packages/app-shell/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@norges-domstoler/app-shell",
+  "version": "1.0.0",
+  "description": "Wrapper for Lovisa applications.",
+  "author": "Elsa team",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/domstolene/designsystem"
+  },
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "homepage": "https://design.domstol.no",
+  "scripts": {
+    "build": "tsup --dts",
+    "build:fast": "tsup",
+    "dev": "pnpm build:fast -- --watch",
+    "typecheck": "tsc --noEmit"
+  },
+  "keywords": [
+    "dds",
+    "app-shell",
+    "design system",
+    "domstolene",
+    "norges domstoler",
+    "elsa"
+  ],
+  "devDependencies": {
+    "@norges-domstoler/storybook-components": "workspace:*",
+    "@storybook/blocks": "8.0.10",
+    "@testing-library/react": "^14.2.2",
+    "@types/react": "^18.2.78",
+    "@types/react-dom": "^18.2.25",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "styled-components": "^6.1.8",
+    "tsup": "^8.0.2",
+    "typescript": "^5.4.5",
+    "vite": "^5.2.8",
+    "vitest": "^1.5.0"
+  },
+  "peerDependencies": {
+    "react": "^16 || ^17 || ^18",
```

Burde sikkert ha dds-components som peer dependency

```suggestion
    "@norges-domstoler/dds-components": ">=15",
    "react": "^16 || ^17 || ^18",
```